edx_arch_experiments/summaryhook_aside/block.py

```diff
@@ -0,0 +1,80 @@
+"""Xblock aside enabling OpenAI driven summariess"""
+
+from django.conf import settings
+from django.template import Context, Template
+from web_fragments.fragment import Fragment
+from xblock.core import XBlockAside
+
+from edx_arch_experiments.summaryhook_aside.waffle import summary_enabled
+
+summary_fragment = """
```

Comment:
Is it common to have this fragment in string format, rather than in a new template file? It's just weird to me to not see this in django html templates